skyrimnet_chatterbox/chatterbox/models/t3/t3_cuda_graphs.py

```
@@ -1,219 +1,219 @@
+
+    def __call__(
+        self,
+        speech_embedding_cache: torch.Tensor,
+        output_logits: torch.Tensor,
+        i_tensor: torch.Tensor,
+        batch_idx: torch.Tensor,
+        speech_pos_embedding_cache: torch.Tensor,
+        generated_ids: torch.Tensor,
+        cfg_weight: float,
+        temperature: float,
+        repetition_penalty_processor: Any = None,
```

Ruff (0.14.10)

Unused method argument: `repetition_penalty_processor`

(ARG002)